schemas/v1.0/schema.yaml

```diff
@@ -0,0 +1,525 @@
+    description: Describes the steps to be taken across one or more APIs to achieve an objective
+    type: object
+    properties:
+      workflowId:
+        description: Unique string to represent the workflow
+        $dynamicAnchor: workflowId
+        type: string
+        pattern: '^[A-Za-z0-9_\-]+$'
+      summary:
+        description: A summary of the purpose or objective of the workflow
+        type: string
+      description:
+        description: A description of the workflow. CommonMark syntax MAY be used for rich text representation
+        type: string
+      inputs:
+        description: A JSON Schema 2020-12 object representing the input parameters used by this workflow
+        $dynamicRef: '#meta'
+      dependsOn:
+        description: A list of workflows that MUST be completed before this workflow can be processed
+        type: array
+        uniqueItems: true
+        items:
+          type: string
+      steps:
+        description: An ordered list of steps where each step represents a call to an API operation or to another workflow
+        type: array
+        uniqueItems: true
+        minItems: 1
+        items:
+          $ref: '#/$defs/step-object'
+      successActions:
+        description: A list of success actions that are applicable for all steps described under this workflow
+        type: array
+        uniqueItems: true
+        minItems: 1
+        items:
+          oneOf:
+            - $ref: '#/$defs/success-action-object'
+            - $ref: '#/$defs/reusable-object'
+      failureActions:
+        description: A list of failure actions that are applicable for all steps described under this workflow
+        type: array
+        uniqueItems: true
+        minItems: 1
+        items:
+          oneOf:
+            - $ref: '#/$defs/failure-action-object'
+            - $ref: '#/$defs/reusable-object'
+      outputs:
+        description: A map between a friendly name and a dynamic output value
+        type: object
+        patternProperties:
+          '^[a-zA-Z0-9\.\-_]+$':
+            type: string
+      parameters:
+        description: A list of parameters that are applicable for all steps described under this workflow
+        type: array
+        uniqueItems: true
+        minItems: 1
+        items:
+          oneOf:
+            - $ref: '#/$defs/parameter-object'
+            - $ref: '#/$defs/reusable-object'
+    required:
+      - workflowId
+      - steps
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  step-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#/$defs/step-object'
+    description: |-
+      Describes a single workflow step which MAY be a call to an
+      API operation (OpenAPI Operation Object or another Workflow Object)
+    type: object
+    properties:
+      stepId:
+        description: Unique string to represent the step
+        $dynamicAnchor: stepId
+        type: string
+        pattern: '^[A-Za-z0-9_\-]+$'
```

Comment:
it is a recommendation so should it be in the schema 

honeslty we would set a MUST it would be nice as well